src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -63,6 +64,34 @@ export class ThankYou extends Component {
         const trackingToken = getUrlParameter(params, 't') || undefined;
         const callerId = getUrlParameter(params, 'c') || undefined;
         this.removeTrackingGetArgs();
+        if (!calledState || !calledNumber) {
+            if (!calledState) {
+                //Send Sentry Breadcrumb
+                Sentry.addBreadcrumb({
+                    category: "Missing Thank You Arguments",
+                    message: "Thank you page accessed without state argument",
+                    level: Sentry.Severity.Info,
+                });
+            }
+            if (!calledNumber) {
+                //Send Sentry Breadcrumb
+                Sentry.addBreadcrumb({
+                    category: "Missing Thank You Arguments",
+                    message: "Thank you page accessed without calledNumber argument",
+                    level: Sentry.Severity.Info,
+                });
+            }
+            //Set Error State
+            this.setState({
+                district: null,
+                callerId,
+                homeDistrictNumber,
+                trackingToken,
+                statsError: Error("Missing argument"),
```

Comment:
Right now I am setting statsError to match the current meta. But in a future pr I think this is obsolete since statsError is defined but never used.